install/froxlor.sql

```diff
@@ -611,6 +611,7 @@ opcache.interned_strings_buffer'),
 	('system', 'documentroot_use_default_value', '0'),
 	('system', 'passwordcryptfunc', '3'),
 	('system', 'axfrservers', ''),
+	('system', 'powerdns_mode', 'Native'),
```

Comment:
You also need to adjust the db_version to `202108180` in this file for new installations

Reply:
OK - thanks for the hint. It is now implemented in: https://github.com/Froxlor/Froxlor/pull/974/commits/29586bd29c2ff4e9b01ac34e57143a353e9aceb3